packages/rocketchat-i18n/i18n/de.i18n.json

```diff
@@ -1904,6 +1948,7 @@
   "Mentions_only": "Nur Erwähnungen",
   "Merge_Channels": "Kanäle zusammenführen",
   "Message_AllowBadWordsFilter": "Wortfilter für Nachrichten verwenden",
+  "Message_AllowConvertLongMessagesToAttachment": "Konvertieren langer Nachrichten in Anhänge zulassen ",
```

Comment:
"Konvertieren langer Nachrichten in Anhänge zulassen "
-> Leerzeichen am Ende gewollt?
-> "in Anhängen"